cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/MapReduceContextConfigTest.java

```diff
@@ -51,23 +75,54 @@ public void testManyMacrosInAppSpec() {
       appCfg.append("${").append(i).append("}");
       hConf.setInt(String.valueOf(i), i);
     }
-    ApplicationSpecification appSpec = new DefaultApplicationSpecification(
-      "name", "desc", appCfg.toString(),
-      new ArtifactId("artifact", new ArtifactVersion("1.0.0"), ArtifactScope.USER),
-      Collections.<String, StreamSpecification>emptyMap(),
-      Collections.<String, String>emptyMap(),
-      Collections.<String, DatasetCreationSpec>emptyMap(),
-      Collections.<String, FlowSpecification>emptyMap(),
-      Collections.<String, MapReduceSpecification>emptyMap(),
-      Collections.<String, SparkSpecification>emptyMap(),
-      Collections.<String, WorkflowSpecification>emptyMap(),
-      Collections.<String, ServiceSpecification>emptyMap(),
-      Collections.<String, ScheduleSpecification>emptyMap(),
-      Collections.<String, ScheduleCreationSpec>emptyMap(),
-      Collections.<String, WorkerSpecification>emptyMap(),
-      Collections.<String, Plugin>emptyMap()
-    );
+    ApplicationSpecification appSpec =
```

Comment:
It looks like the changes to this method are just indentation. If so, please change it back so it's clear there are no changes for the reviewer.